app/src/androidTest/java/org/simple/clinic/sync/BaseSyncCoordinatorAndroidTest.kt

```diff
@@ -1,75 +0,0 @@
-package org.simple.clinic.sync
-
-import com.f2prateek.rx.preferences2.Preference
-import com.google.common.truth.Truth.assertThat
-import io.reactivex.Completable
-import io.reactivex.Single
-import org.junit.Test
-import org.simple.clinic.patient.SyncStatus
-import org.simple.clinic.patient.SyncStatus.DONE
-import org.simple.clinic.patient.SyncStatus.PENDING
-import org.simple.clinic.util.None
-import org.simple.clinic.util.Optional
-
-// TODO: Use this class for all data sync tests.
-abstract class BaseSyncCoordinatorAndroidTest<T, P> {
```

Comment:
Why are we removing this? 🤔

Reply:
Because the tests were badly written and deeply coupled to the implementation, they added no value. What these tests were meant to cover are now being handled by the new integration tests we have added.

Leaving these in will get in the way when refactoring, so I removed them because we don't need them anymore.